Source/WebCore/css/DOMCSSNamespace.cpp

```diff
@@ -58,9 +58,16 @@ static String valueWithoutImportant(const String& value)
 
 bool DOMCSSNamespace::supports(Document& document, const String& property, const String& value)
 {
-    CSSPropertyID propertyID = cssPropertyID(property.stripWhiteSpace());
-
     CSSParserContext parserContext(document);
+
+    auto propertyNameWithoutWhitespace = property.stripWhiteSpace();
+    CSSPropertyID propertyID = cssPropertyID(propertyNameWithoutWhitespace);
+    if (propertyID == CSSPropertyInvalid && isCustomPropertyName(propertyNameWithoutWhitespace)) {
+        auto dummyStyle = MutableStyleProperties::create();
+        constexpr bool important = false;
```

Comment:
I suggest calling this variable either `importance` or `notImportant`, to make the next line more readable. Currently, it says `important`, which is the exact opposite of what it is.

Reply:
Good point. Will do.